.github/workflows/ci.yml

```diff
@@ -38,10 +38,10 @@ jobs:
     - name: Format with isort
       run: |
         isort --check --diff .
-    - name: Setup node.js (for pyright)
-      uses: actions/setup-node@v1
-      with:
-        node-version: "12"
+#    - name: Setup node.js (for pyright)
```

Comment:
Can we use pyright like we do now in palkit?

Reply:
only if you are willing to make it pass in the first place.

Reply:
Oh right, there's that problem with the sample classes